### PR TITLE
minor edits to readme

### PR DESCRIPTION
**Pull request by CyberFlameGO**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+Nacho can now be used in production environments with some degree of stability.
 
 If you find any bugs, please create an issue or contact us in the [Discord server](https://discord.gg/ewcYeERKJw)!
 
 **NachoSpigot supports Java 8 to Java 16!**
 
 ## Download
 **Stable:** [https://nacho.sculas.xyz/](https://nacho.sculas.xyz/)
-**Development:** [https://nachodev.sculas.xyz/](https://nachodev.sculas.xyz/)
+**Development:** Use `git` or GitHub's `gh` CLI to clone the repository, or [Download as ZIP](https://github.com/CobbleSword/NachoSpigot/archive/refs/heads/master.zip)
```
Comment (Elierrr) [inline README.md:17]: Not needed

Comment (CyberFlameGO) [reply README.md:17]: Why use a third-party site to achieve the same thing you can within GitHub

Comment (Elierrr) [reply README.md:17]: I'm saying that we should just remove the line

Comment (Lucaskyy) [reply README.md:17]: We can just remove the line. And what you did there was a source download, where the devbuild link actually provides a devbuild.

Comment (CyberFlameGO) [reply README.md:17]: My bad, I was thrown off by the zip file being downloaded when I opened the URL. Didn't actually bother to extract it and see for myself

**Review by Elierrr — APPROVED** [review]

**Comment by Lucaskyy** [comment]
LGTM!

**Pull request merged**